packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -252,11 +252,12 @@ export class EpochTracker implements IPersistedFileCache {
                 // This will only throw if it is an epoch error.
                 this.checkForEpochErrorCore(epochFromResponse, error.errorMessage);
             } catch (epochError) {
-                annotateError(epochError, {
-                    fromCache,
-                    clientEpoch: this.fluidEpoch,
-                    fetchType,
-                });
+                annotateErrorObject(epochError,
```

Comment:
in the future please avoid mixing functional change and refactoring, it just makes PRs larger and harder to follow